ui/src/components/harvest-games/HarvestGamesPage.js

```diff
@@ -276,7 +289,7 @@ const HarvestGames = (props) => {
                 >
                   Leaderboard
                 </Tab>
-                <Tab
+                {/* <Tab
```

Comment:
can you just delete the Pre-Games and Leaderboard sections and make sure just "Leaderboard" (which contains the Live Leaderboard) and "Game Instructions" are shown in both web and mobile?